roles/dtc/common/templates/ndfc_vrf_lite.j2

```diff
@@ -1,6 +1,6 @@
 {% if item.ospf is defined and item.ospf %}
-{% include 'ndfc_vrf_lite_ospf.j2' %}
+  {% include 'ndfc_vrf_lite_ospf.j2' %}
```

Comment:
How does the indentation of line 2 here make the rendered configuration look? Is the indentation impactful at all as we've left-aligned Jinja2 in most other files if not mistaken.

Reply:
Thanks @juburnet! Can you resolve the conversations as you complete them as it is a check done in GitHub that needs to be done to enable merging the PR.